src/AndroidDebugLauncher/Telemetry.cs

```diff
@@ -40,6 +40,11 @@ public enum LaunchFailureCode
             RunAsFailure,
 
             /// <summary>
+            /// run-as returned 'Package 'bla' is not debuggable'
```

Comment:
what is bla? be careful with works like blah and bla as they translate into swear words in russian